README.md

```diff
@@ -184,6 +184,49 @@ All labels except `M-failed-staging-checks`, `M-failed-staging-other`,
 them useful when determining the current state of a PR.
 
 
+## Commit metadata in PR description
+
+Pull request description may contain a header and/or a trailer paragraphs with
+special `name: value` metadata fields documented below. Header fields are
+recognized only by the bot. Some trailer fields are recognized by the bot,
+GitHub, and/or git. When assembling the commit message, Anubis strips the
+header but keeps the trailer. Pull requests with descriptions containing
+invalid metadata are labeled `M-failed-description` and are not merged.
+
+```
+Authored-by: Actual Author <user@host>
+
+Regular PR description paragraph(s), if any, go here...
+
+Co-authored-by: Co-Author One <user1@host1>
+Co-authored-by: Co-Author Two <user2@host2>
+```
+
+* `Authored-by: Author Name <user@host>`: A custom commit author. Add this
+  field to the PR description header when Anubis should not use the pull
+  request author as the commit author. Mentioning this (or similarly spelled)
+  fields elsewhere in the PR description should trigger an
+  `M-failed-description` violation. At most one `Authored-by` field is
+  permitted. Anubis removes the header paragraph when assembling the commit
+  message because neither GitHub nor git recognize this field and because the
+  information will be preserved as the commit author field.
+
+* `Co-authored-by: Another Author Name <other@host>`: A commit co-author. This
+  field documents additional commit authors. It is recognized by
+  [GitHub](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors)
+  and, to an extent, [git](https://git-scm.com/docs/git-interpret-trailers).
+  Multiple unique fields are permitted.
```

Comment:
If it is not too difficult, we should reject PRs with identical trailer fields. If it is too difficult, we should drop the word "unique" from this paragraph.

Reply:
>  reject PRs with identical trailer fields

Done (e7f9c7e).